scripts/pcm-dpc/covid-19/preprocess.py

```diff
@@ -24,6 +26,10 @@ def preprocess(self):
 
     def generate_tmcf(self):
         """Generate the template mcf."""
+        out = self.name + '.tmcf'
+        if os.path.exists(out):
+            os.remove(out)
+
```

Comment:
Perhaps it would be better for the class to have the invariant that it only produces the tMCF file once, relying on its own logic, not on the filesystem.  This will make it clearer that the file contents are the same for every run.

E.g., it is not clear that self.geo_template() always produces the same output.

Reply:
I did what you suggested below